src/algebra/order/with_zero.lean

```diff
@@ -207,6 +207,10 @@ variables [linear_ordered_comm_group_with_zero α]
 lemma zero_lt_one₀ : (0 : α) < 1 :=
 lt_of_le_of_ne zero_le_one zero_ne_one
 
+lemma mul_le_one₀ (ha : a ≤ 1) (hb : b ≤ 1) : a * b ≤ 1 := mul_le_one' ha hb
+
+lemma one_le_mul₀ (ha : 1 ≤ a) (hb : 1 ≤ b) : 1 ≤ a * b := one_le_mul ha hb
```

Comment:
Wait, why do you need those lemmas? We have enough lemmas called `mul_le_something` so I would rather it not be even more confusing!

Reply:
This is already in mathlib - I will merge the latest version.